### PR TITLE
Drop old Symfony code

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targeting this branch, because this is BC.

### Subject

Removing old code that never gets executed because we dropped SF 2.3 to 2.7

### CODE REVIEW

File: src/Builder/DatagridBuilder.php [hunk 1]
```diff
@@ -126,8 +128,7 @@ public function addFilter(DatagridInterface $datagrid, $type, FieldDescriptionIn
 
         $fieldDescription->mergeOption('field_options', ['required' => false]);
 
-        // NEXT_MAJOR: Remove first check (when requirement of Symfony is >= 2.8)
```
Comment (OskarStark) [inline src/Builder/DatagridBuilder.php]: which check was removed?

Comment (jordisala1991) [reply src/Builder/DatagridBuilder.php]: None, it cannot be removed, will add this comment back

Comment (jordisala1991) [reply src/Builder/DatagridBuilder.php]: Fixed!

**Comment by greg0ire** [comment]
Thanks @jordisala1991 !

**Pull request merged**